### ISSUE TITLE
Does BitLocker still work after recover site database

### ISSUE THREAD

**Issue by Li-xianhua** [MicrosoftDocs/memdocs#1901]
I have a question: 
we are using MECM 2103 BitLocker management, I found that BitLocker recovery key was encrypted by default in site database, I don't know which certificate is being used. So I want to know, if I recover site database to another server, do I lost all the encrypted recovery keys? 

thanks!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 4e3ff295-17b1-94ea-94c4-9a5eaeb82a4d
* Version Independent ID: 5e593199-9a69-1b19-9fb7-2853d98c04a6
* Content: [Site recovery - Configuration Manager](https://docs.microsoft.com/en-us/mem/configmgr/core/servers/manage/recover-sites)
* Content Source: [memdocs/configmgr/core/servers/manage/recover-sites.md](https://github.com/MicrosoftDocs/memdocs/blob/master/memdocs/configmgr/core/servers/manage/recover-sites.md)
* Product: **configuration-manager**
* Technology: **configmgr-core**
* GitHub Login: @mestew
* Microsoft Alias: **mstewart**

**Comment by aczechowski** [comment]
I'll add this information to the docs, the changes will publish with 2111 release.

Basically, you need to use the following steps to backup and then restore the certificate:

- https://docs.microsoft.com/mem/configmgr/protect/deploy-use/bitlocker/encrypt-recovery-data#back-up-certificate
- https://docs.microsoft.com/mem/configmgr/protect/deploy-use/bitlocker/encrypt-recovery-data#restore-certificate

**Comment by Li-xianhua** [comment]
Thanks very much, But we haven't install BitLockerManagement_Cert to SCCM database, and there is no certificate to backup.
Even more, tried: SELECT * from sys.certificates on SCCM database, returns nothing...


what's the default cert SCCM using to encrypt BitLocker recovery key...




------------------&nbsp;Original&nbsp;------------------
From:                                                                                                                        "MicrosoftDocs/memdocs"                                                                                    ***@***.***&gt;;
Date:&nbsp;Wed, Nov 24, 2021 01:03 PM
***@***.***&gt;;
***@***.******@***.***&gt;;
Subject:&nbsp;Re: [MicrosoftDocs/memdocs] Does BitLocker still work after recover site database (#1901)





 
I'll add this information to the docs, the changes will publish with 2111 release.
 
Basically, you need to use the following steps to backup and then restore the certificate:
  
https://docs.microsoft.com/mem/configmgr/protect/deploy-use/bitlocker/encrypt-recovery-data#back-up-certificate
 
https://docs.microsoft.com/mem/configmgr/protect/deploy-use/bitlocker/encrypt-recovery-data#restore-certificate
  
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by aczechowski** [comment]
I’m sorry that you’re having problems, and I want to make sure it gets to the right people that can help. There are a couple of options available to provide the best level of support for this:

- Post on [Configuration Manager forums](https://docs.microsoft.com/answers/products/mem). These are moderated by our community MVPs who are very experienced, knowledgeable, and helpful.
- Use [Microsoft Support](https://aka.ms/cmcbsupport) to search for the issue, or open a case with professional support.
- Contact your Microsoft Premier team, such as a Premier Field Engineer or Technical Account Manager.

**Comment by Li-xianhua** [comment]
thanks all the same
